app/client/views/resident/activities/roles/roles.js

```diff
@@ -11,54 +11,56 @@ Template.residentFacilitatorRolesChart.onRendered(function () {
 
     // Add 'facilitatorRoleName' field to each activity,
     // Containing Role name
-    activities = _.map(activities, function (activity) {
+    activities = _.map(activities, function(activity) {
       // Get activity type name via collection helper
       activity.facilitatorRoleName = activity.facilitatorRole();
 
       return activity;
     });
 
     // Group activities by facilitator role
-    const facilitatorRoleCounts = d3.nest()
-      .key(function (activity) {
+    const facilitatorRoleCounts = d3
+      .nest()
+      .key(function(activity) {
         return activity.facilitatorRoleName;
       })
-      .rollup(function (facilitatorRole) { return facilitatorRole.length })
+      .rollup(function(facilitatorRole) {
+        return facilitatorRole.length;
+      })
       .entries(activities);
 
-      // Get reference to chart container
-      const svg = dimple.newSvg("#facilitatorRolesChart", "100%", "100%");
-
-      // Initialize the activity type chart
-      const facilitatorRolesChart = new dimple.chart(svg, facilitatorRoleCounts);
-
-      // Set chart boundaries based on parent container size
-      facilitatorRolesChart.setBounds("20%", "5%", "75%", "60%");
-
-      // Add facilitator roles to x axis
-      const xAxis = facilitatorRolesChart.addMeasureAxis("x", "values");
-
-      // Set x axis title
-      const xAxisTitle = TAPi18n.__("residentFacilitatorRolesChart-xAxis-title");
-      xAxis.title = xAxisTitle;
-
-      // Disable grid lines
-      xAxis.showGridlines = false;
-
-      // Add facilitator role counts to y axis
-      const yAxis = facilitatorRolesChart.addCategoryAxis("y", "key");
-
-      // Set y axis title
-      const yAxisTitle = TAPi18n.__("residentFacilitatorRolesChart-yAxis-title");
-      yAxis.title = yAxisTitle;
-
-      // Disable grid lines
-      yAxis.showGridlines = false;
-
-      // Add bar plot
-      facilitatorRolesChart.addSeries(null, dimple.plot.bar);
-
-      // Render chart
-      facilitatorRolesChart.draw();
+    if (facilitatorRoleCounts && facilitatorRoleCounts.length > 0) {
+      const data = [
+        {
+          type: "bar",
+          y: _.map(facilitatorRoleCounts, r => r.key.substr(0, 6)),
+          x: _.map(facilitatorRoleCounts, r => r.values),
+          text: _.map(
+            facilitatorRoleCounts,
+            r => `key:${r.key} \n value:${r.values}`
+          ),
+          marker: {
+            color: "#9ac0db"
+          },
+          width: 0.5,
+          orientation: "h"
+        }
+      ];
+
+      const layout = {
+        xaxis: {
+          title: TAPi18n.__("residentFacilitatorRolesChart-xAxis-title")
+        },
+        yaxis: {
+          title: TAPi18n.__("residentFacilitatorRolesChart-yAxis-title")
+        },
+        width: 600,
```

Comment:
I think we can enable responsive: true on this chart, so it will scale based on viewport size.